services/src/messaging/src/main/java/org/openkilda/messaging/payload/flow/FlowReroutePayload.java

```diff
@@ -15,20 +15,41 @@
 
 package org.openkilda.messaging.payload.flow;
 
-import com.fasterxml.jackson.annotation.JsonProperty;
+import org.openkilda.messaging.Utils;
 import org.openkilda.messaging.info.event.PathInfoData;
 
-public class FlowReroutePayload extends FlowPathPayload {
```

Comment:
Class `PathInfoData`, that uses as a field type in `FlowPathPayload`, is not available from `api` package now. It's needs to add dependency on `messaging` or move all transitive used classes to `api` package

Reply:
Didn't get, why for you need any messaging object inside api module? "convertation" is done inside NB. NB have access to both - messaging and api.

Reply:
We had [`FlowPathPayload`]( https://github.com/telstra/open-kilda/blob/7436ef20606f7f6ae6f80ba3bdd551ed02c98484/services/src/messaging/src/main/java/org/openkilda/messaging/payload/flow/FlowPathPayload.java) class. You suggested me to use MapStruct [mapper](https://github.com/telstra/open-kilda/blob/365a37a8832ab3412ec058024909b79752f1e4d4/services/src/northbound/src/main/java/org/openkilda/northbound/converter/FlowMapper.java#L37) instead of [manual converter](https://github.com/telstra/open-kilda/blob/a43528ffd17e5029e3c138a33c7606a8db74fea7/services/src/messaging/src/main/java/org/openkilda/messaging/payload/flow/FlowPayloadToFlowConverter.java#L92) to generate it from "conversation" payload. Ok, but this avoids FlowPathPayload's `@Value` immutability and this conversation starts. After first message in this thread, FlowPathPayload was renamed into FlowPathOutput and moved to `api` package, but it still holds [fields](https://github.com/telstra/open-kilda/blob/365a37a8832ab3412ec058024909b79752f1e4d4/services/src/api/src/main/java/org/openkilda/northbound/dto/flows/FlowPathOutput.java#L52) that depends directly on `messaging` module. 
Show me where I misunderstand you

Reply:
I think that we should rollback FlowPathPayload's MapStruct mapping (include moving to `api` and making it mutable) for now

Reply:
So, rollbacking all changes corresponds with MapStruct mapping